### PR TITLE
Add explanation for "CN"

### PR DESCRIPTION
**Pull request by furukawa3**
This PR adds explanation for "CN" #16456

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -572,7 +572,7 @@ checked. WARNING: do **not** reuse a CA that is used in a different context unle
 the risks and the mechanisms to protect the CA's usage.
 
 * `--requestheader-client-ca-file` Required. PEM-encoded certificate bundle. A valid client certificate must be presented and validated against the certificate authorities in the specified file before the request headers are checked for user names.
-* `--requestheader-allowed-names` Optional.  List of common names (cn). If set, a valid client certificate with a Common Name (cn) in the specified list must be presented before the request headers are checked for user names. If empty, any Common Name is allowed.
+* `--requestheader-allowed-names` Optional.  List of Common Name(CN)s. If set, a valid client certificate with a CN in the specified list must be presented before the request headers are checked for user names. If empty, any CN is allowed.
```
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/authentication.md]: How about:
```suggestion
* `--requestheader-allowed-names` Optional.  List of Common Name values (CNs). If set, a valid client certificate with a CN in the specified list must be presented before the request headers are checked for user names. If empty, any CN is allowed.
```
? (and also for `content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md`)
Does that read better?

**Comment by furukawa3** [comment]
/assign @makoscafee
Hi, makoscafee,  could you plz review my patch ??

File: content/en/docs/reference/access-authn-authz/authentication.md (hunk 1 above)
Comment (furukawa3) [reply content/en/docs/reference/access-authn-authz/authentication.md]: LGTM!!  I'll fix it.

**Comment by furukawa3** [comment]
> /lgtm
> @furukawa3 if you wanted to tweak [/docs/concepts/cluster-administration/certificates/](https://kubernetes.io/docs/concepts/cluster-administration/certificates/) in the same PR, that would work too.

I will !!  Thanks for your point.

**Comment by furukawa3** [comment]
I found another "CN" without explanation here: [content/en/docs/reference/setup-tools/kubeadm/implementation-details.md](https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/)  Could you also check this page?

File: content/en/docs/concepts/cluster-administration/certificates.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ manually through `easyrsa`, `openssl` or `cfssl`.
         tar xzf easy-rsa.tar.gz
         cd easy-rsa-master/easyrsa3
         ./easyrsa init-pki
-1.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default CN to use.)
+1.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default Common Name (CN) to use.)
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/certificates.md]: This line has nested brackets. Fine in code; odd in English text.
How about:
```suggestion
1.  Generate a new certificate authority (CA). `--batch` sets automatic mode; `--req-cn` specifies the Common Name (CN) for the CA's new root certificate.
```

Also consider adding a newline after “certificate authority (CA).”.

Comment (furukawa3) [reply content/en/docs/concepts/cluster-administration/certificates.md]: Thanks for your suggestion.  I'll fix it.

File: content/en/docs/concepts/services-networking/ingress.md [hunk 3]
```diff
@@ -319,8 +319,8 @@ type: kubernetes.io/tls
 
 Referencing this secret in an Ingress tells the Ingress controller to
 secure the channel from the client to the load balancer using TLS. You need to make
-sure the TLS secret you created came from a certificate that contains a CN
-for `sslexample.foo.com`.
+sure the TLS secret you created came from a certificate that contains a Common
+Name (CN), aka Fully Qualified Domain Name (FQDN) for `sslexample.foo.com`.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/ingress.md]: “aka” might count as [jargon](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-jargon-and-idioms); however, I'm inclined to accept this PR and leave room for future iteration if anyone has the time and inclination.

Comment (zacharysarah) [reply content/en/docs/concepts/services-networking/ingress.md]: I'm more fine with an extraneous commit (like what I'm about to make) than I am with leaving jargon in a doc.

Comment (furukawa3) [reply content/en/docs/concepts/services-networking/ingress.md]: Thanks @sftim and @zacharysarah .  That's good to update jargon :)

**Pull request merged**